openapi/paths/quotes@{id}@timeline@{messageId}.yaml

```
@@ -0,0 +1,50 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: messageId
+    in: path
+    description: ID of the quote timeline message.
+    required: true
+    schema:
+      type: string
+get:
+  x-products:
+    - Core
+  tags:
+    - Quotes timeline
+  summary: Retrieve a quote timeline message
+  operationId: GetQuoteTimelineMessage
+  x-sdk-operation-name: getTimelineMessage
+  description: Retrieves a quote message with a specified ID.
```

nitpick: we should use 'quote timeline message' for all instances
```suggestion
  description: Retrieves a quote timeline message with a specified ID.
```